### ISSUE TITLE
Design Quarter

### ISSUE THREAD

**Issue by brentgreyling** [decentraland/districts#23]
Core hub for Modern Design and Visual Identity in VR.
Center of Excellence for Virtual Arts from concept to completed construction.

**UPDATED:** _To expand on my initial thoughts, as requested_

Title: **Design Quarter**
Description: **Core hub for Design and Visual Identity in VR.
Center of Excellence for Virtual Arts from concept to completed construction.**
Project Lead: **None required**, I would be happy to facilitate or represent, anyone is welcome to comment here (constructively please) or contact me directly using my github username at gmail.
Project Team : **None required**
Goal: **To simply designate/indicate a commercial area of common interest related to visual identity, design and art.**
As pointed out in the official roadmap visual identity and design are crucial to the growth of Decentraland (and Social VR in general). In VR, architecture and engineering is relatively unrestrained and might even be considered a function of design, or at the very least more closely related. Creating a commercially focused hive with it's value centered around design is sure to increase the standard of visual representation and identity overall... as well as to create a "nice little district" both to visit and trade in.
Ownership: As in life.... **Free market, anyone can buy and own property.** As in Decentralised VR.... Nobody should be required to manage or regulate other than perhaps to enforce global and common sense measures against unannounced publically offensive actions - Material considered offensive to carry an upfront warning as regulated by the higher authority that makes up Decentraland.
Matters of squatting and speculating also to be deferred to same higher authority.
LAND Needed: **As much as is made available by Decentraland for interested parties to purchase.**
Same model as the second ring BUT where land is **INITIALLY sold at a fixed price of 1000 MANA**.
Thereby offering the increased overall benefit afforded by initially allowing equal opportunity to the strongest creatives, not the wealthiest or most established ones. Any hierarchy would develop naturally after establishment.
How will funds be raised: **No funds need to be raised.** Simple initial sale and development of land. Resale of land and business at market determined values related to company value and position.
How will contributors see a return value: **An area of common interest in visual identity, art and design would draw clients requiring those types of services as well as businesses that offer related auxillary and support services.** Creative neighbours can benefit by drawing visitors, sharing inspiration and leveraging on each others strengths while overall development, healthy competition and collaboration is encouraged.
Attachments / Designs / Budgets : **NONE at present.**

In summary... Earmark a commercial district geared toward design. Let it start fairly and equally and it should grow to the benefit of all.

_Perhaps a little idealistic, I suspect, but still a good district to hope and aim for. Thank you for partaking._

**Comment by RodneyGraham** [comment]
Great idea

**Comment by dclancy13** [comment]
That's a great idea! For consistency and to allow contributors to asses projects equally, could you include at the beginning of your proposal a filled in "District Proposal Framework"
Thanks so much for your contribution!

**District Proposal Framework**

Title:  **Name of Project**
Description: **Discuss in a paragraph the details of your Proposal**
Project Lead: **Name and Email**
Project Team :**Name / Email / Roles of each committed team member**
(list no more than 5 members)
 Goal:  **What does this District want to accomplish (i.e. generate revenue for its owners, act as a public gathering place, just be for fun) ** 
Ownership: **How will ownership of the district be distributed?**
LAND Needed: **How Much LAND will be required to successfully launch this District**
Startup Costs: **How much (in ETH) will be needed to fund building the District (pay for developers / designers / marketing)**
How will funds be raised: **Explain how startup costs will be raised to complete and launch the District**
How will contributors see a return value: **Explain why others would want to participate in this District**
Attachments / Designs / Budgets : Upload any supporting files.

Many District proposals may be too early on in the development process to answer all of these questions but the idea is to establish a common rubric for submitting a proposal, to get contributors to start thinking about how they intend to execute on their projects and to give potential supporters an idea of how far along in the development process each particular proposal is. This will help them make a well informed decision in choosing which project to participate in.

**Comment by nkranders** [comment]
I am a 3D artist and architectural technologist.Contact me if you need 3d modelling.
